doc: add pricing and plans doc for en-US (#52)
zeabur/docs: pages/billing/plans.en-US.mdx

---
title: Plans
ogImageTitle: Plans
ogImageSubtitle: Different plans on Zeabur
---

# Plans

Zeabur has multiple plans to fit your need.  

Each plan has a different set of features and limits. 

## Free Plan

Free plan is designed to help you evaluate Zeabur or are fit for hobbyist projects. You can get all features on Zeabur.

> Free plan gets **$4** of credits that do not accumulate.

Reviewer: ```suggestion
Free plan gets **$5** of credits per month that do not accumulate.
```

Once you have run out of credits of the month, your deployments will be suspended until the next month, where you have to manually restart it.

### Tier Offering

- Max to **512 MB** of RAM
- Max to **1 vCPU** capacity
- Max to **1 GB** of storage

> - **$4** of monthly usage

Reviewer: ```suggestion
- **$5** of monthly usage
```

- Max to **3** projects or **8** services
- Public documentation and community answers support
- Need to redeploy projects after the new monthly credit is applied to your account

## Developer Plan

You can upgrade to the **Developer Plan** by paying $5 per month through your credit card, Alipay or other payments methods.

### Tier Offering

- Max to **8 GB** of RAM
- Max to **4 vCPU** capacity
- Max to **100 GB** of storage

> - **$11** of monthly usage

Reviewer: ```suggestion
- **$10** of monthly usage
```

- Technical team priority support
- No projects or services limits

## Team Plan

Team plan is designed to provide the most resources for all your scaled needs,  you can upgrade to the **Team Plan** by paying $80 per month.

### Tier Offering

- Max to **32 GB** of RAM
- Max to **8 vCPU** capacity

> - Max to **1 T** of storage

Reviewer: ```suggestion
- Max to **1 TB** of storage
```
